### ISSUE TITLE
A guide to install with new Docker container?

### ISSUE THREAD

**Issue by CampNowhere2021** [jkirkcaldy/plex-utills#57]
I have read the readme, reddit posts, and the blog post but none of them give detailed directions how to actually install this especially with a NAS such as Synology.  I have Docker installed on my Synology NAS and found this repo in the registry and downloaded the image but I have no clue how to actually install and run this, let alone how to mount to the correct folder and edit the config.  

Will there be any detailed guides on how to install this on a NAS using Docker?

**Comment by jkirkcaldy** [comment]
I’m writing the documentation at the moment. Though I don’t have much experience with synology. I may have to see if I can get an xpenology vm going to see how it works.   
  

  

  
  
  
    
  

  
  
>

**Comment by CampNowhere2021** [comment]
That would be great to have a guide.  This is how Docker looks on a Synology NAS, it's a nice GUI and I found the image for plex utils on there and started the container but I have no idea how to access the plex-utils web gui to set anything up or if I need to change any values on my setup.
![d1](https://user-images.githubusercontent.com/94004614/141132359-22bba313-8371-4090-abb9-6dea3fb9c997.jpg)

![d2](https://user-images.githubusercontent.com/94004614/141132369-949d3f1d-34f1-4065-8d84-7742727b5ba8.jpg)

**Comment by CampNowhere2021** [comment]
These are all the setting that can be changed once the plex-utils image is downloaded from Docker.  Not sure what all the settings need to be or still how to access the web gui.
![d2](https://user-images.githubusercontent.com/94004614/141143667-b1a01c9c-99c4-402a-be91-c9a263f5d164.jpg
![d3](https://user-images.githubusercontent.com/94004614/141143703-4b891c35-49cd-485a-b533-1b877d29ade9.jpg)
)
![d4](https://user-images.githubusercontent.com/94004614/141143713-86884e7f-d785-42a2-bd1c-85300050da53.jpg)
![d5](https://user-images.githubusercontent.com/94004614/141143723-05c59c9b-7199-402b-aa70-536a91ebbaf2.jpg)
![d6](https://user-images.githubusercontent.com/94004614/141143736-36a83ac8-a44f-4c15-ac50-f8176fe0b186.jpg)
![d7](https://user-images.githubusercontent.com/94004614/141143747-0e9d3d24-f6d5-4e60-8290-d36121639e0f.jpg)

![d8](https://user-images.githubusercontent.com/94004614/141143759-6199e0fb-f09f-4b85-ab5d-278071a5613a.jpg)
![d9](https://user-images.githubusercontent.com/94004614/141143769-1728eaf6-4616-4fa1-95c8-aaf91f217242.jpg)

**Comment by jkirkcaldy** [comment]
Ok. 

So to start with, you will need to add three entries in the Volumes tab for the following mount paths:
/config <- this should point so somewhere on your array where you want to store the config database
/logs <- this should point to somewhere on your array wehre you want to store the logs

I created a `plex-utills` folder on my array with a folder for the config and a folder for the logs inside that to keep it together

/films <- this should point to where all your plex media lives on your array. 

You will then need to modify the TZ line in the Environment tab. This should match your location. 

I'm not sure how the Synology boxes handle permissions so you can try it with those steps and see if it works. 

You have the ports set to auto, so to get to the ui you would go to the IP address of your synology box + the port number that has been assigned.

**Comment by CampNowhere2021** [comment]
This is what I have typed into the Volumes tab, not sure if that is correct but I did create a plex-utils folder then a subfolder for config and logs.  
![d1](https://user-images.githubusercontent.com/94004614/141180725-b4d9d355-872e-40d5-955a-69a7969f8caf.jpg)


I cannot access the web ui at all.  I type in the ip of my nas and tried all 3 of the ports listed and the web ui does not load.  It just times out.

So for example the ip I tried all 3: 
MYIP:89167 container port 443
MYIP:89168 container port 5000
MYIP:89169 container port 80
